docs/developer-docs/latest/development/backend-customization/models.md

```diff
@@ -494,6 +495,34 @@ The `tableName` key defines the name of the join table. It has to be specified o
 
 :::::
 
+#### Custom fields
+
+[Custom fields](/developer-docs/latest/development/custom-field.md) extend Strapi’s capabilities by adding new types of fields to content-types. Custom fields are explicitly defined in the [attributes](#model-attributes) of a model with `type: customField`.
+Custom fields attributes also accept:
+
+- the `customField` additional parameter to mention the registered custom field to use, using:
```

Comment:
Maybe it is just me, but I don't understand this sentence.

Reply:
Thanks for the feedback.

Basically, in a model's schema file, a custom field has the following specificities:
- `type: customField`
- a `customField` attribute which indicates which custom field to use (i.e. which registered custom field should be used). Its value follows:
        - either the `plugin::plugin-name.field-name` format if a plugin created the custom field 
        - or the `global::field-name` format for a custom field "specific to the current Strapi application" 
(technically, "if not created by a plugin, but not only…" but I didn't want to make the description even more confusing with edge cases)

What do you think if I rewrite it like this? 👇
```suggestion
Custom fields' attributes also show the following specificities:
-a `customField` attribute which acts as a unique identifier to indicate which registered custom field should be used. Its value follows:
   - either the `plugin::plugin-name.field-name` format if a plugin created the custom field 
   - or the `global::field-name` format for a custom field specific to the current Strapi application.
```

cc @meganelacheny as I know you found this part confusing as well 👀

Reply:
Much clearer! Thank you @pwizla 😊